Cargo.toml

```diff
@@ -10,7 +10,7 @@ categories = ["data-structures"]
 edition = "2018"
 
 [build-dependencies]
-rustc_version = "0.4"
+rustc_version = "1.37"
```

Comment:
I don't think this is necessary at all anymore?

Reply:
Since it's only for `build.rs` and that got deleted.

Reply:
Good point. Fixed in https://github.com/softdevteam/vob/pull/74/commits/44c967c1755c0abdb19956b09a9191c2d1e62c6c.